src/main/java/com/example/gamelist/GameMapper.java

```diff
@@ -2,12 +2,23 @@
 
 import org.apache.ibatis.annotations.Mapper;
 import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
 
 import java.util.List;
+import java.util.Optional;
+
 
 @Mapper
 public interface GameMapper {
 
+    //データを一括で取得する。　GET
     @Select("SELECT * FROM gamelists")
     List<Game> findAll();
+
+    @Select("SELECT * FROM gamelists WHERE id = #{id}")
+    Optional<Game> findById(int id);
+
+    @Update("UPDATE gamelists SET name = {#name} WHERE id = #{id}")
+    void updateGamelist(Game game);
```

Comment:
Gamelist -> Gameにしましょう。
```suggestion
    void updateGame(Game game);
```
もしくは
```suggestion
    void update(Game game);
```
でよいかと。

Reply:
2b74f1cea15d19e140d50897b9d7ee35956a724a

こちらはGameではなく、updateに変更しました。

Reply:
OKです。